ui-components/src/Theming/Theming.scss

```diff
@@ -0,0 +1,30 @@
+.mdx-table {
```

Comment:
At the moment, Storybook adds a ton of styles that override cell content to a vanilla mdx table that it appears you can only get out of by adding a custom Storybook theme, so this table just lets us have richer & predictable cell content / React cell content.